projects/README.md

```diff
@@ -0,0 +1,26 @@
+# GradleUp Projects
+
+The key projects are listed below.
+You can find the full list of projects being developed by the team
+on the [GitHub organization](https://github.com/GradleUp)
+
+## Tools for Android
+
+- [Auto Manifest](../auto-manifest/README.md) -
+  Generates `AndroidManifest.xml` in simple libraries so that you don't have to
+- [Gradle static analysis plugin](../static-analysis-plugin/README.md) -
+  A Gradle plugin to easily apply the same setup of static analysis tools
+  across different Android, Java or Kotlin projects.
+- [Gr8](../gr8/README.md) -
+  Gr8 makes it easy to shadow, shrink, and minimize your jars.
+
+## Gradle DevX
+
+- [Gratatouille](../gratatouille/README.md) -
+  Gratatouille is an opinionated framework to build Gradle plugins. Write pure Kotlin functions and the Gratatouille KSP processor generates tasks, workers and wiring code for you.
+- [New Maven Central Publishing](../nmcp/README.md) -
```

Comment:
Nit I usually refer to it as just "nmcp", I find it shorter and easier to remember:

```suggestion
- [NMCP](../nmcp/README.md) (New Maven Central Publishing) -
```